todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsViewModel.kt

```diff
@@ -34,38 +38,66 @@ import com.example.android.architecture.blueprints.todoapp.data.source.TasksRepo
  * whereas the [Bindable] getters allow us to add some logic to it. This is
  * preferable to having logic in the XML layout.
  */
-class StatisticsViewModel(
-        private val context: Application,
-        private val tasksRepository: TasksRepository
-) : AndroidViewModel(context) {
-
-    val dataLoading = ObservableBoolean(false)
-    val error = ObservableBoolean(false)
-    val numberOfActiveTasksString = ObservableField<String>()
-    val numberOfCompletedTasksString = ObservableField<String>()
+class StatisticsViewModel(context: Application, private val tasksRepository: TasksRepository) :
+    AndroidViewModel(context) {
+
+    private val _dataLoading = MutableLiveData<Boolean>()
```

Comment:
why do we have all of these new lines? is this a standard I don't know of?

Reply:
We've always done this this way. I don't think we have specific style guidelines on it.